stores/transaction/src/main/java/com/bloxbean/cardano/yaci/store/transaction/TransactionStoreProperties.java

```diff
@@ -0,0 +1,22 @@
+package com.bloxbean.cardano.yaci.store.transaction;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class TransactionStoreProperties {
+
+    @Builder.Default
+    private boolean pruningEnabled = false;
+
+    @Builder.Default
+    private int pruningInterval = 144000;
+
+    @Builder.Default
+    private int pruningSafeSlot = 43200; // 2160 blocks
```

Comment:
I think @matiwinnetou  was referring to this value. 
But let's keep it as it is, as we are using similar value in other stores and it's configurable.